hadoop-hdds/interface-admin/src/main/proto/ScmAdminProtocol.proto

```diff
@@ -607,6 +610,38 @@ message ContainerBalancerStatusResponseProto {
   required bool isRunning = 1;
 }
 
+message ContainerBalancerStatusInfoRequestProto {
+  optional string traceID = 1;
+}
+
+message ContainerBalancerStatusInfoResponseProto {
+  optional bool isRunning = 1;
+  optional ContainerBalancerStatusInfo containerBalancerStatusInfo = 2;
+}
+message ContainerBalancerStatusInfo {
+  optional uint64 startedAt = 1;
+  optional ContainerBalancerConfigurationProto configuration = 2;
+  repeated ContainerBalancerTaskIterationStatusInfo iterationsStatusInfo = 3;
+}
+
+message ContainerBalancerTaskIterationStatusInfo {
+  optional int32 iterationNumber = 1;
+  optional string iterationResult = 2;
+  optional int64 sizeScheduledForMove = 3;
+  optional int64 dataSizeMovedGB = 4;
```

Comment:
Nit: `sizeScheduledForMove` can be standardized to include the units (similar to `dataSizeMovedGB`). This also applies to other size related attributes like `NodeTransferInfo`, `sizeEnteringNodes`, and `sizeLeavingNodes`.